project/Build.scala

```diff
@@ -12,7 +12,10 @@ object BuildSettings {
   val buildScalaVersion = "2.12.6"
 
   val buildSettings = Defaults.defaultSettings ++ Seq(
-    scalacOptions ++= Seq("-encoding", "UTF-8", "-Xlint", "-deprecation", "-unchecked", "-feature"),
+    scalacOptions ++= (CrossVersion.partialVersion(scalaVersion.value) match {
+      case Some((2,  11)) =>  Seq("-encoding", "UTF-8", "-Xlint", "-deprecation", "-unchecked", "-feature", "-target:jvm-1.6")
```

Comment:
Regarding our offline discussion, does it make sense to have support for Java 1.6?